### ISSUE TITLE
Include sample files

### ISSUE THREAD

**Issue by kenlck** [lasanthaS/ShortTermLoiteringDetection#1]
@lasanthaS I could not run the code, as it requires the sample file

**Comment by lasanthaS** [comment]
Yes, this requires some sample files. For the samples, I am using **PETS2007** dataset which you can find at [1]. Within that, there are few sample scenarios which each contains 999 images. To run the application you need to point the application to that location and run. You can set the path to the dataset at [2].

Please note that this is still a work in progress. Hence some of the scenarios are not covered and may not function properly. 

[1] http://www.cvg.reading.ac.uk/PETS2007/data.html
[2] https://github.com/lasanthaS/ShortTermLoiteringDetection/blob/master/utils/Constants.py#L4

**Comment by kenlck** [comment]
Alright, thank you for the info!

**Comment by Guepardow** [comment]
Hi, as the data of PETS2007 seem to be unreachable on the official website, I am wondering if you still have them. If you have a backup, could you please share them through a copy on Drive or Dropbox ? Thank you

**Comment by la-tuna** [comment]
Hi!
Could you share the PETS2007 dataset? Please
